app.rb

```diff
@@ -1,17 +1,22 @@
 require './student'
 require './teacher'
+require './loading'
+require './person'
 require './rental'
 require './book'
+require 'json'
 
 class App
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, kindly handle the case when one or more of the files are missing. You could use this line of code `return [] unless File.exists?("people.json")`.